frontend/packages/ceph-storage-plugin/src/components/block-pool/dashbaord/mirroring-card-item.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
+
+export const MirroringCardItem: React.FC<MirroringCardItemProps> = React.memo(
+  ({ title, isLoading = false, children, error = false, valueClassName, errorMessage }) => {
+    const { t } = useTranslation();
+
+    let status: React.ReactNode;
+
+    if (error) {
+      status = (
+        <span className="text-secondary">
+          {errorMessage || t('ceph-storage-plugin~Not available')}
+        </span>
+      );
+    } else if (isLoading) {
+      status = <div className="skeleton-text" />;
+    } else {
+      status = children;
+    }
+    return (
+      <>
+        {title && <dt className="ceph-block-pool-mirroring-card__item-title">{title}</dt>}
```

Comment:
I coped form <DetailItem> components, only difference is for mirroring card sometime i need title and sometime not